src/Support/Compatibility/MiddlewareBridge.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Nuwave\Lighthouse\Support\Compatibility;
+
+interface MiddlewareBridge
+{
+    /**
+     * Get all of the defined middleware short-hand names.
+     *
+     * @return array
```

Comment:
Change the return type in docs to a more specific one. If this is just an array of strings use `string[]`

Reply:
Wasn't even aware this was a valid type hint in php-doc :)